### PR TITLE
Update hotswitch from 1.20 to 1.21

### PR DESCRIPTION
**Pull request by ran-dall**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Created with [`cask-repair`](https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask).

### CODE REVIEW

**Comment by core-code** [comment]
`Error: It seems the App source '/usr/local/Caskroom/hotswitch/1.21/HotSwitch.app' is not there.
`

**Review by core-code — DISMISSED** [review]

**Comment by core-code** [comment]
```
- app "HotSwitch.app"
+ app "Users/ao/Downloads/HotSwitch 2020-09-14 03-48-40/HotSwitch.app"
```

@oniatsu this looks like a definitive bug in the packaging of the latest release

File: Casks/hotswitch.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  sha256 "ee1c0d171d1726ab2771408c1dc0e8427e0eeed0e110150e9f000e4dbf3a6801"
 
   url "https://oniatsu.github.io/HotSwitch/release/zip/HotSwitch.zip"
   appcast "https://github.com/oniatsu/HotSwitch/releases.atom"
   name "HotSwitch"
   desc "Fast window switcher using a 2-stroke hotkey"
   homepage "https://oniatsu.github.io/HotSwitch/"
 
-  app "HotSwitch.app"
+  app "Users/ao/Downloads/HotSwitch 2020-09-14 03-48-40/HotSwitch.app"
```
Comment (reitermarkus) [inline Casks/hotswitch.rb:11]: Pinging @oniatsu since this definitely seems to be unintentional.

**Comment by reitermarkus** [comment]
Let's wait a bit for a response from upstream.

File: Casks/hotswitch.rb (hunk 1 above)
Comment (oniatsu) [reply Casks/hotswitch.rb:11]: I fixed the HotSwitch.zip content.

@ran-dall Check it, please.

**Comment by core-code** [comment]
@oniatsu thanks everything seems to be fine now

**Review by core-code — APPROVED** [review]

**Pull request merged**